ouroboros-network/src/Ouroboros/Network/Block.hs

```diff
@@ -46,11 +50,8 @@ module Ouroboros.Network.Block (
   , encodeChainHash
   , decodePoint
   , decodeChainHash
-
-  , pattern GenesisPoint
-  , pattern BlockPoint
-  , atSlot
-  , withHash
+    -- * Serialised block/header
+  , Serialised(..)
```

Comment:
I renamed it from `SerialisedBlock` to `Serialised`, because I will reuse it for headers too.